mercadopago/px-ios
# (Added)[PXN-4171] Added confirmButton, pay flow and notifications in ReviewAndConfirm

## What has changed?

Added pay flow in Review And Confirm with a new confirmButton that inherits from our PXAnimatedButton (class we use to set up all our loading buttons). 
- Changes in PXReviewAndConfirmViewController, its viewModel, callbacks to OneTapFlow and childViewController inits and deinits. 

Added _PX -> Integrator notifications_, to inform on certain events occurring and allowing the integrator to optionally call a method on each of them.

## How to test:

- Add a builder integration with an Advanced Configuration, setting _CustomReviewAndConfirmViewController_ as the injected VC. 
- Check the flow: **OneTap -> Review And Confirm -> Congrats**

## Prints for layout changes:

| CC | AM |
|:-:|:-:|
|![Simulator Screen Recording - iPhone 13 - 2022-06-16 at 10 07 21](https://user-images.githubusercontent.com/61248081/174082417-d158defa-d8fd-4790-99bd-dd128151d2ab.gif)  |  ![Simulator Screen Recording - iPhone 13 - 2022-06-16 at 10 06 42](https://user-images.githubusercontent.com/61248081/174082692-d7bf072d-64a0-4c46-b3fc-dc1a1c8190dc.gif)
|
